backend/src/bundles/password-reset/helpers/create-password-reset-ink.ts

```diff
@@ -0,0 +1,10 @@
+type UserData = {
+    userId: number | string;
+    token: string;
+};
+
+const createPasswordResetLink = ({ userId, token }: UserData): string => {
+    return `http://localhost:3000/reset-password/${userId}/${token}`;
```

Comment:
This will only work locally but we also have deploy

Reply:
And is passing token directly in url string works okay without errors?

Reply:
And also is it really necessary to include userId in path since we store it in token

Reply:
> And also is it really necessary to include userId in path since we store it in token

Yes, because I implemented one-use token, with this purpose it is added the current password of user and without userId its impossible to verify token, because of leak of password-hash property of user

Reply:
I meant that you can parse the jwt token back into an object and get userId from it on frontend and it will be the same userId that you've included in url